src/main/resources/logback.xml

```diff
@@ -9,7 +9,21 @@
         </encoder>
     </appender>
 
-    <root level="info">
-        <appender-ref ref="STDOUT" />
+    <appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="STDOUT"/>
+    </appender>
+
+    <logger name="ch.jbert" level="DEBUG"/>
+    <logger name="ch.jbert.event" level="DEBUG"/>
+    <logger name="ch.jbert.gpio" level="DEBUG"/>
+    <logger name="ch.jbert.mpd" level="DEBUG"/>
+    <logger name="ch.jbert.rfid" level="DEBUG"/>
+    <logger name="ch.jbert.util" level="DEBUG"/>
```

Comment:
If all the sub-packages have the same log level, declaration of just the `ch.jbert` logger should suffice.

Reply:
Right. But it sucks to change it on the target...